os_net_config/impl_nmstate.py

```diff
@@ -50,6 +50,27 @@
 # Import the raw NetConfig object so we can call its methods
 netconfig = os_net_config.NetConfig()
 
+_VF_BIND_DRV_SCRIPT = '''
+dpdk_vfs="{dpdk_vfs}"
+linux_vfs="{linux_vfs}"
+set -x
+
+for vfid in $dpdk_vfs $linux_vfs; do
+    vf_pci_id=$(basename $(readlink /sys/class/net/$1/device/virtfn$vfid)) &&
+    modalias=$(cat /sys/class/net/$1/device/virtfn$vfid/modalias) &&
+    def_driver=$(modprobe -R $modalias) &&
+    if echo $dpdk_vfs | grep -qw $vfid && ! [ "$def_driver" = mlx5_core ]; then
+        driver=vfio-pci
+    else
+        driver=$def_driver
+    fi &&
```

Comment:
Even if no paths here should contain any spaces nor special chars, you need to quote everything.

```bash
    modalias=$(cat "/sys/class/net/$1/device/virtfn$vfid/modalias") &&
    def_driver=$(modprobe -R "$modalias") &&
    if echo "$dpdk_vfs" | grep -qw "$vfid" && ! [ "$def_driver" = mlx5_core ]; then
        driver=vfio-pci
    else
        driver="$def_driver"
    fi &&
```

Reply:
Done